### ISSUE TITLE
Wrong auxclasspath if project is stored outside of workspace

### ISSUE THREAD

**Issue by demonfiddler** [pmd/pmd-eclipse-plugin#96]
**Rule Set:** Type Resolution

**Description:** The CloneMethodMustImplementCloneable rule can erroneously report a violation when the class in question extends one that _does_ implement Cloneable.

**Code Sample demonstrating the issue:**

```java
class MyArrayList<T> extends ArrayList<T> {
	// PMD.CloneMethodMustImplementCloneable incorrectly flags this method as a violation.
	@Override
	public Object clone() {
		throw new UnsupportedOperationException();
	}
}
```

**Running PMD through:** Eclipse

**Comment by adangel** [comment]
When the project was stored outside of the workspace (e.g. when the project is imported, but not copied into the workspace), then the paths for the auxclasspath where wrong - they were always relative to the workspace location.
